app/scripts/scripts.js

```diff
@@ -1,17 +1,28 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
+(function () {
```

Comment:
Actually I'm confused on how we test this.
My browser says it can't find the bower plugins we included in index.html and it's preventing my scripts.js from running.